kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -44,20 +44,29 @@ export interface TopicsListProps {
   areTopicsFetching: boolean;
   topics: TopicWithDetailedInfo[];
   totalPages: number;
+
```

Comment:
why is there extra line additions here if the code did not change , in all of this file.

Reply:
Fixed